examples/java/spring-boot/src/main/java/uk/gov/api/springboot/config/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.api.springboot.config;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import uk.gov.api.models.metadata.ErrorResponse;
+import uk.gov.api.springboot.exceptions.CorrelationIdMalformedException;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+  @ExceptionHandler(CorrelationIdMalformedException.class)
+  protected ResponseEntity<ErrorResponse> handleCorrelationIdMalformedException(
+      CorrelationIdMalformedException ex, WebRequest request) {
+    ErrorResponse errorResponse = new ErrorResponse();
```

Comment:
this is a V1 error response, and does not negotiate, so will cause us issues in the past as we add future versions :thinking: